lightningd/connect_control.c

```diff
@@ -291,11 +312,23 @@ static void try_connect(const tal_t *ctx,
 	struct delayed_reconnect *d;
 	struct peer *peer;
 
+	/* Don't stack, unless this is an instant reconnect */
+	d = delayed_reconnect_map_get(ld->delayed_reconnect_map, id);
+	if (d) {
+		if (seconds_delay) {
+			log_peer_debug(ld->log, id, "Already reconnecting");
+			return;
+		}
```

Comment:
```suggestion
		if (seconds_delay) {
			log_peer_debug(ld->log, id, "Already reconnecting");
			return;
		}
```
should we call `tal_free(d);` also here?

Reply:
No, d is the existing timer!  We're leaving that alone...